modules/sticky-table-headers/main/index.coffee

```diff
@@ -63,7 +62,18 @@ getChildren = (parent, selector, single) ->
 createStickyHeaderNodes = (real, cloned) ->
   for i in [0...real.length]
     cloneEvents(real[i], cloned[i])
+    # The real table shouldn't show the sticky header nodes
     hx.select(real[i]).classed('hx-sticky-table-invisible', true)
+    hx.select(cloned[i]).classed('hx-sticky-table-invisible', false)
+
+cloneTableAndNodeEvents = (selection, realTable, tableClone, body, single) ->
+  innerTableClone = selection.append tableClone.clone(true)
+  # We make all the cells invisible by default and then only make the ones we care about visible in `createStickyHeaderNodes`
+  innerTableClone.selectAll('th, td').classed('hx-sticky-table-invisible', true)
+  realNodes = getChildrenFromTable realTable, body, single
+  clonedNodes = getChildrenFromTable innerTableClone, body, single
```

Comment:
I attempted to replace `getChildrenFromTable` with `shallowSelect`/`shallowSelectAll` but it caused issues with events not being cloned properly (possibly only in `PhantomJS`):

```coffee
getChildrenFromCompoundSelector = (table, selectorChain) ->
  selectorChain.reduce (sel, [selector, single]) ->
    sel[if single then 'shallowSelect' else 'shallowSelectAll'](selector)
  , table
```
Use:
```coffee
getChildrenFromCompoundSelector(table, [['thead', true], ['tr'], ['th, td']])
```

There must be a better way of doing `getChildrenFromTable`...